src/main.c

```diff
@@ -22,7 +23,20 @@ int main (){
     internal_clock();
 
     init_all();
-    
+    // void adjust_priorities();
```

Comment:
make another commit with a cleaned main function body (maintain the original main.c structure)